src/test/codegen/async-fn-debug.rs

```diff
@@ -3,7 +3,6 @@
 //  - The generator types and variants are marked artificial
 //  - Captured vars from the source are not marked artificial
 //
-// ignore-tidy-linelength
```

Comment:
The easiest fix here might be to just keep the line as `//` so line numbers do not move around.

Reply:
Good idea!